engines/gob/detection/tables_adibou2.h

```diff
@@ -72,7 +72,8 @@
 	{
 		"adibou2",
 		"ADIBOU 2.10 Environnement",
-		AD_ENTRY1s("intro.stk", "f2b797819aeedee557e904b0b5ccd82e", 8736454),
+		AD_ENTRY2s("intro.stk", "f2b797819aeedee557e904b0b5ccd82e", 8736454,
+				"BECBF210.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```

Comment:
I don't understand, why is this change needed, since intro.stk has a unique MD5 hash?